ubiquity/stepDellBootstrap.ui

```diff
@@ -616,6 +616,30 @@
                     <property name="position">3</property>
                   </packing>
                 </child>
+                <child>
+                  <object class="GtkAlignment" id="alignment13">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <property name="left_padding">25</property>
+                    <child>
+                      <object class="GtkLabel" id="raid_warning">
+                        <property name="width_request">600</property>
+                        <property name="visible">True</property>
+                        <property name="can_focus">False</property>
+			<property name="label" translatable="yes">Disk with * in the front is part of a raid.</property>
```

Comment:
Slight wording tweak: `Are part of a RAID array`.

Reply:
modify accordingly.